Agilefreaks/FreaksCatalog-API
# [180] Add validations for FreakCreate

## Description
In this PR we add validation for norm_id, role_id and level_id. We can't create a freak with association norm, level and role that not exist.
## Related

[[180] Add validations for FreakCreate](https://trello.com/c/ZLhPJNPW/180-add-validations-for-freakcreate)

## Steps to Test or Reproduce
Go in Insomnia and add this mutation to create a freak:
```
mutation($norm_id: ID!, $role_id: ID!, $level_id: ID!) {
	freakCreate(
		input: {
			firstName: "john",
			lastName: "doe",
			description: "ceva",
			email: "freak@gmail.com",
			normId: $norm_id,
			roleId: $role_id,
			levelId: $level_id
		}
	) {
		firstName
		lastName
		description
		email
        norm {
            name
        }
		role {
			name
		}
		level{
			name
		}
	}
}
```
Query variables:
```{
"norm_id": 1,
  "role_id": 1,
  "level_id": 1
}
```
If you add some value for this three parameters that is not in db you will see an error
